src/applications/find-forms/containers/SearchResults.jsx

```diff
@@ -149,6 +161,11 @@ export class SearchResults extends Component {
       );
     }
 
+    // Derive sort options
+    const sortOptions = useLighthouseSearchAlgo
+      ? [TEST_OPTION_RELEVANCE, ...SORT_OPTIONS]
+      : SORT_OPTIONS;
```

Comment:
Maybe call this constant `DEFAULT_SORT_OPTIONS` instead of `SORT_OPTIONS`?

Reply:
Ah great point sir!